tests-e2e/cypress/helpers/rest-utils-ts.ts

```diff
@@ -0,0 +1,103 @@
+import axios from "axios";
+
+export class RestUtils {
+    config: any;
+
+    constructor(conf: any) {
+        this.config = conf;
+    }
+
+    private async getToken() {
+        const params = {
+            headers: {
+                'Content-Type': 'application/x-www-form-urlencoded'
+            }
+        };
+        const data = 'grant_type=client_credentials&client_id=' + this.config.env.CLIENT_ID + '&client_secret=' + this.config.env.CLIENT_SECRET;
+        const response = await axios.post(this.config.env.TOKEN_URL, data, params);
+        return response.data.access_token;
+    }
+
+    public async getData(url: string, params: any = {}) {
+        try {
+            params = this.setToken(params, await this.getToken());
+            const response = await axios.get(url, params);
+            // console.log('get Data Status', response.status);
+            // console.log('get Data StatusText', response.statusText);
+            // console.log('get Data Res', minify(response.data));
```

Comment:
minor commented out console.logs?

Reply:
Yes left these on purpose; they're useful for debugging but muddy up the console quite a bit during a run.

Reply:
sounds good